src/test/java/it/gov/pagopa/payhub/pdnd/anpr/controller/PdndServiceControllerTest.java

```diff
@@ -27,7 +27,7 @@ class PdndServiceControllerTest {
     private PdndService pdndService;
 
     @Test
-    void testAnprServiceE002CitizenGet_Success() throws Exception {
+    void testCitizenDataByFiscalCodeGet_Success() throws Exception {
```

Comment:
Use conventional naming for test given<GIVEN>When<CONDITION>Then<RESULTEXPECTED>

Reply:
modified